### PR TITLE
🐛 Source Amazon Seller Partner: updates type to allow double point precision.

### LINKED ISSUE #17163: Source Amazon Seller Partner: invalid input for double precision
This Github issue is synchronized with Zendesk:

**Ticket ID:** [#2386](https://airbyte7538.zendesk.com/agent/tickets/2386)
**Priority:** normal
**Group:** Community Assistance Engineer
**Assignee:** Marcos Marx

**Original ticket description:**

<blockquote><div class="zd-comment" dir="auto"><ul style="margin: 0 0 0 10px; padding: 0 0 0 20px;" dir="auto">
<li style="padding-bottom: 10px;">Is this your first time deploying Airbyte?: <strong>Yes</strong>
</li>
<li style="padding-bottom: 10px;">OS Version / Instance: <strong>Windows</strong>
</li>
<li style="padding-bottom: 10px;">Memory / Disk:</li>
<li style="padding-bottom: 10px;">Deployment:  <strong>Local Docker Containers</strong>
</li>
<li style="padding-bottom: 10px;">Airbyte Version: <strong>0.40.4</strong>
</li>
<li style="padding-bottom: 10px;">Source name/version:  <strong>Amazon Seller Partner</strong>
</li>
<li style="padding-bottom: 10px;">Destination name/version: <strong>Postgres 15.0 or MYSQL 8.0.30</strong>
</li>
<li style="padding-bottom: 10px;">Step: <strong>The issue is happening during sync</strong>
</li>
<li style="padding-bottom: 10px;">Description: <strong>I tried to normalize the amazon seller partner data in a mysql database first and then moved to postgress for another try. I can fetch the data in raw json with not problem but i need them normalized.</strong>
</li>
</ul>
<p dir="auto">This is the main error in logs:</p>
<pre style="max-width: 694px; word-wrap: break-word;"><code style="background-color: #f9f9f9; display: block; overflow: auto; padding: 2px 5px;">21 of 56 ERROR creating table model public.listfinancialeventgroups_beginningbalance.................................... [ERROR in 0.19s]
22 of 56 ERROR creating table model public.listfinancialeventgroups_convertedtotal...................................... [ERROR in 0.20s]
23 of 56 ERROR creating table model public.listfinancialeventgroups_originaltotal....................................... [ERROR in 0.18s]
Database Error in model get_merchant_listings_all_data (models/generated/airbyte_tables/public/get_merchant_listings_all_data.sql)
invalid input syntax for type double precision: ""
compiled SQL at ../build/run/airbyte_utils/models/generated/airbyte_tables/public/get_merchant_listings_all_data.sql
Database Error in model listfinancialeventgroups_beginningbalance (models/generated/airbyte_tables/public/listfinancialeventgroups_beginningbalance.sql)
invalid input syntax for type bigint: "0.0"
compiled SQL at ../build/run/airbyte_utils/models/generated/airbyte_tables/public/listfinancialeventgroups_beginningbalance.sql
Database Error in model listfinancialeventgroups_convertedtotal (models/generated/airbyte_tables/public/listfinancialeventgroups_convertedtotal.sql)
invalid input syntax for type bigint: "693.09"
compiled SQL at ../build/run/airbyte_utils/models/generated/airbyte_tables/public/listfinancialeventgroups_convertedtotal.sql
Database Error in model listfinancialeventgroups_originaltotal (models/generated/airbyte_tables/public/listfinancialeventgroups_originaltotal.sql)
</code></pre>
<p dir="auto"><a href="https://discuss.airbyte.io/uploads/short-url/v5P0knf0z5KDldfVRLrTEZseuEg.txt" style="color: #006699; font-weight: bold; text-decoration: none;" rel="noreferrer">logs-142.txt</a> (278.7 KB)</p> 

[<a href="https://discuss.airbyte.io/t/invalid-input-syntax-for-type-normalization-asp-postgres-mysql/2670/1" rel="noreferrer">Discourse post</a>]</div></blockquote>

### PR DESCRIPTION
**Pull request by natalyjazzviolin**
## What
Fixes https://github.com/airbytehq/airbyte/issues/17163

## How
Changes type from `integer` to `number` to allow double point precision.

## Recommended reading order
1. `airbyte-integrations/connectors/source-amazon-seller-partner/source_amazon_seller_partner/schemas/ListFinancialEventGroups.json`

## 🚨 User Impact 🚨
Are there any breaking changes? What is the end result perceived by the user? If yes, please merge this PR with the 🚨🚨 emoji so changelog authors can further highlight this if needed.

## Pre-merge Checklist

<summary><strong>Updating a connector</strong></summary>

### Community member or Airbyter

- [ ] Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- [ ] Secrets in the connector's spec are annotated with `airbyte_secret`
- [ ] Unit & integration tests added and passing. Community members, please provide proof of success locally e.g: screenshot or copy-paste unit, integration, and acceptance test output. To run acceptance tests for a Python connector, follow instructions in the README. For java connectors run `./gradlew :airbyte-integrations:connectors:<name>:integrationTest`.
- [ ] Code reviews completed
- [ ] Documentation updated
    - [ ] Connector's `README.md`
    - [ ] Connector's `bootstrap.md`. See [description and examples](https://docs.google.com/document/d/1ypdgmwmEHWv-TrO4_YOQ7pAJGVrMp5BOkEVh831N260/edit?usp=sharing)
    - [ ] Changelog updated in `docs/integrations/<source or destination>/<name>.md` including changelog. See changelog [example](https://docs.airbyte.io/integrations/sources/stripe#changelog)
- [ ] PR name follows [PR naming conventions](https://docs.airbyte.com/contributing-to-airbyte/issues-and-pull-requests)

## Tests

<details><summary><strong>Unit</strong></summary>

*Put your unit tests output here.*

</details>

<details><summary><strong>Integration</strong></summary>

*Put your integration tests output here.*

</details>

<details><summary><strong>Acceptance</strong></summary>

*Put your acceptance tests output here.*

</details>

### CODE REVIEW

**Review by marcosmarxm — APPROVED** [review]
Thanks! I hope tests pass 🤞

File: airbyte-integrations/connectors/source-amazon-seller-partner/acceptance-test-config.yml [hunk 1]
```diff
@@ -2,6 +2,7 @@ connector_image: airbyte/source-amazon-seller-partner:dev
 tests:
   spec:
     - spec_path: "integration_tests/spec.json"
+    - backward_compatibility_tests_config.disable_for_version: "0.2.26"
```
Comment (marcosmarxm) [inline airbyte-integrations/connectors/source-amazon-seller-partner/acceptance-test-config.yml:5]: This isn't the correct place to put. @natalyjazzviolin search other examples of how add the `backward_compatibility_tests_config` probably will be there one for discovery

**Review by marcosmarxm — CHANGES_REQUESTED** [review]

File: airbyte-integrations/connectors/source-amazon-seller-partner/acceptance-test-config.yml (hunk 1 above)
Comment (natalyjazzviolin) [reply airbyte-integrations/connectors/source-amazon-seller-partner/acceptance-test-config.yml]: Yup! Realized my mistake!

**Review by marcosmarxm — APPROVED** [review]

**Pull request merged**